src/til/ut_til/MathTests.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+#include "til/math.h"
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+class MathTests
+{
+    TEST_CLASS(MathTests);
+
+    template<typename TG, typename TX = ptrdiff_t>
+    struct TestCase
+    {
+        TG given;
+        TX expected;
+    };
+
+    template<class TilMath, typename TG, typename TX, int N>
+    static void _RunCases(TilMath, const std::array<TestCase<TG, TX>, N>& cases)
+    {
+        for (const auto& tc : cases)
+        {
+            VERIFY_ARE_EQUAL(tc.expected, TilMath::template cast<decltype(tc.expected)>(tc.given));
+        }
+    }
+
+    TEST_METHOD(Truncating)
+    {
+        std::array<TestCase<long double, ptrdiff_t>, 8> cases{
+            TestCase<long double, ptrdiff_t>{ 1., 1 },
+            { 1.9, 1 },
+            { -7.1, -7 },
+            { -8.5, -8 },
+            { PTRDIFF_MAX + 0.5, PTRDIFF_MAX },
+            { PTRDIFF_MIN - 0.5, PTRDIFF_MIN },
+            { INFINITY, PTRDIFF_MAX },
+            { -INFINITY, PTRDIFF_MIN },
+        };

```

Comment:
I appreciate your code coverage here through the use of this creative test case permutation.

However, one of my goals with the til tests was to show people how to actually use it.

Can you add one more case that shows a trivial way of using it correctly as reference source for future copy/paste?

Reply:
Oops. Nevermind me. I thought this was the point file. Ignore this.